damus/en-US.lproj/Localizable.stringsdict

```diff
@@ -12,10 +12,8 @@
 			<string>NSStringPluralRuleType</string>
 			<key>NSStringFormatValueTypeKey</key>
 			<string>d</string>
-			<key>zero</key>
```

Comment:
`zero` is optional in English, and we can omit it if the string is the same as `other`.